docs/jwst/jump/arguments.rst

```diff
@@ -1,7 +1,39 @@
 Arguments
 =========
 
-The ``jump`` step has one optional argument that can be set by the user:
+The ``jump`` step has five optional arguments that can be set by the user:
 
 * ``--rejection_threshold``: A floating-point value that sets the sigma
-  threshold for jump detection.
+  threshold for jump detection. This sigma is determined using the read noise from the
```

Comment:
Is this a description of how the default value is set?  As written, it appears to be instructions for how the user should determine the value.

Reply:
I reworded the paragraph to make the distinction between the sigma calculated in the code and the rejection_threshold setting.